### PR TITLE
Routes setup

### PR DESCRIPTION
**Pull request by ntarn**
Add Maps API and setup Java Servlet Backend for the setup of the custom routes visualization tool

### CODE REVIEW

File: custom-routes-vis/.factorypath [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<factorypath>
```
Comment (ChenyuHeidiZhang) [inline custom-routes-vis/.factorypath:1]: what is this file and why do we need it?

File: custom-routes-vis/.project [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ChenyuHeidiZhang) [inline custom-routes-vis/.project:1]: do we need this file?

File: custom-routes-vis/.settings/org.eclipse.core.resources.prefs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```
Comment (ChenyuHeidiZhang) [inline custom-routes-vis/.settings/org.eclipse.core.resources.prefs:1]: do we need this file?

File: custom-routes-vis/.settings/org.eclipse.jdt.apt.core.prefs [hunk 4]
```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
```
Comment (ChenyuHeidiZhang) [inline custom-routes-vis/.settings/org.eclipse.jdt.apt.core.prefs:1]: I don't think we need those files. Are you using eclipse? Maybe add them to gitignore

File: custom-routes-vis/.settings/org.eclipse.jdt.core.prefs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+eclipse.preferences.version=1
```
Comment (ChenyuHeidiZhang) [inline custom-routes-vis/.settings/org.eclipse.jdt.core.prefs:1]: eclipse file^

File: custom-routes-vis/.settings/org.eclipse.m2e.core.prefs [hunk 6]
```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```
Comment (ChenyuHeidiZhang) [inline custom-routes-vis/.settings/org.eclipse.m2e.core.prefs:1]: ^

File: custom-routes-vis/README.md [hunk 7]
```diff
@@ -0,0 +1,2 @@
+This web application visualizes custom routes selection on a map and will be used for testing by the NavSDK team.
+It utilizes the Maps API and Java Servlets as the backend.
```
Comment (carmenBen) [inline custom-routes-vis/README.md:2]: Include instructions on how to run this.

**Pull request merged**